packages/lib/src/index.ts

```diff
@@ -87,7 +87,11 @@ export function isComponentType<T = React.ReactElement>(
     element != null &&
     typeof element === 'object' &&
     'type' in element &&
-    (element.type as any).displayName === displayName
+    ((element.type as any).displayName === displayName ||
```

Comment:
reinforces for me that we should have a more elegant/clever way of checking a component's type (out of scope here)